web/source/dom/domEventHandlers.ts

```diff
@@ -474,9 +476,14 @@ namespace com.keyman.dom {
       var osk = this.keyman.osk;
 
       var Levent = PreProcessor._GetKeyEventProperties(e, false);
-      if(Levent == null || !osk.ready) {
+      if(Levent == null) {
         return true;
       }
+      /*
+      if(!osk) {
+        return true;
+      }
+      */
```

Comment:
This is commented out -- should it be removed?

Reply:
Reminder-to-self:  remove.  (For when I 'batch' the final changes needed.  No need to overwork the BAs.)